Formula/no-more-secrets.rb

```diff
@@ -0,0 +1,16 @@
+class NoMoreSecrets < Formula
+  desc 'Recreates the SETEC ASTRONOMY effect from "Sneakers"'
```

Comment:
I wasn't suggesting to use italics (and definitely am for sticking with plain text in the description), merely trying to say that the capitalized title sufficiently sticks out from the sentence case that is used in the rest of the description. Of course that's just my subjective feeling and as a non-native speaker it's difficult to judge how this is perceived by others.